projects/kit/components/input-phone/input-phone.component.ts

```diff
@@ -96,14 +96,21 @@ export class TuiInputPhoneComponent
                           .split('')
                           .map(item => (item === '#' ? /\d/ : item)),
                   ],
-        pipe: value => {
+        pipe: (value, config) => {
             if (this.allowText) {
                 return value;
             }
 
-            return value === '' && this.focused && !this.readOnly
-                ? `${this.countryCode} `
-                : value.replace(/-$/, '');
+            if (value === '' && this.focused && !this.readOnly) {
```

Comment:
I think it's better to just reset native input to empty value upon leaving the control. You can set `nativeFocusableElement.value = ''` or create a setter for `get nativeValue` and do it in `onActiveZone` at the bottom of it.